juju/juju
# Update EC2 region data

## Update EC2 region data

- Adds af-south-1
- Adds eu-south-1
- Fixes "AWS GovCloud (US-West)" price listing.
- Adds missing data for me-south-1 and ap-east-1

## QA steps

```
$ juju update-public-clouds
Fetching latest public cloud list...
Updated list of public clouds on this client, 2 cloud regions added:

    added cloud region:
        - aws/af-south-1
        - aws/eu-south-1
```

```
$ juju bootstrap aws/af-south-1
Creating Juju controller "aws-af-south-1" on aws/af-south-1
Looking for packaged Juju agent version 2.8-rc1 for amd64
No packaged binary found, preparing local Juju agent binary
Launching controller instance(s) on aws/af-south-1...
 - i-09cdca1d373ec6871 (arch=amd64 mem=4G cores=2)af-south-1a"
Installing Juju agent on bootstrap instance
Fetching Juju GUI 2.15.3
Waiting for address
Attempting to connect to 13.245.28.166:22
Attempting to connect to 172.31.13.248:22
Connected to 13.245.28.166
Running machine configuration script...
Bootstrap agent now started
Contacting Juju controller at 13.245.28.166 to verify accessibility...

Bootstrap complete, controller "aws-af-south-1" is now available
Controller machines are in the "controller" model
Initial model "default" added
```

```
$ juju bootstrap aws/eu-south-1
Creating Juju controller "aws-eu-south-1" on aws/eu-south-1
Looking for packaged Juju agent version 2.8-rc1 for amd64
No packaged binary found, preparing local Juju agent binary
Launching controller instance(s) on aws/eu-south-1...
 - i-0319fcbb10ccd73fa (arch=amd64 mem=4G cores=2)eu-south-1a"
Installing Juju agent on bootstrap instance
Fetching Juju GUI 2.15.3
Waiting for address
Attempting to connect to 15.161.148.0:22
Attempting to connect to 172.31.34.180:22
Connected to 15.161.148.0
Running machine configuration script...
Bootstrap agent now started
Contacting Juju controller at 15.161.148.0 to verify accessibility...

Bootstrap complete, controller "aws-eu-south-1" is now available
Controller machines are in the "controller" model
Initial model "default" added
```

## Documentation changes

N/A

## Bug reference

https://bugs.launchpad.net/juju/+bug/1872137
